images/ubuntu/scripts/build/configure-apt-sources.sh

```diff
@@ -8,17 +8,17 @@ source $HELPER_SCRIPTS/os.sh
 
 touch /etc/apt/apt-mirrors.txt
 
-printf "http://azure.archive.ubuntu.com/ubuntu/\tpriority:1\n" | tee -a /etc/apt/apt-mirrors.txt
-printf "http://archive.ubuntu.com/ubuntu/\tpriority:2\n" | tee -a /etc/apt/apt-mirrors.txt
-printf "http://security.ubuntu.com/ubuntu/\tpriority:3\n" | tee -a /etc/apt/apt-mirrors.txt
+printf "https://azure.archive.ubuntu.com/ubuntu/\tpriority:1\n" | tee -a /etc/apt/apt-mirrors.txt
```

Comment:
I think this url is not available 🤔

Reply:
It would be *really* nice if Azure would offer it. When people use https://github.com/marketplace/actions/harden-runner, it breaks things:
https://github.com/ben-manes/caffeine/actions/runs/14483252724/job/40624089687